src/audio/mixer.c

```diff
@@ -77,7 +78,7 @@ static void mix_n_s16(struct comp_dev *dev, struct comp_buffer *sink,
 
 			dest = buffer_write_frag_s16(sink, frag);
 
-			/* Saturate to 16 bits */
+		/* Saturate to 16 bits */
```

Comment:
What happened to this comment? It should be aligned with the code (I hope GH doesn't mess up the layout though)

Reply:
GH is write, it was some unintended change, good catch